client/my-sites/marketing/do-it-for-me/difm-landing.tsx

```diff
@@ -285,6 +285,7 @@ export default function DIFMLanding( {
 	siteId?: number | null;
 	isStoreFlow: boolean;
 } ) {
+	const required_product_slugs = [ PLAN_PREMIUM, WPCOM_DIFM_LITE, PLAN_BUSINESS ];
```

Comment:
```suggestion
	const requiredProductSlugs = [ PLAN_PREMIUM, WPCOM_DIFM_LITE, PLAN_BUSINESS ];
```

This should be in camel case. :)